merlin/server/server_util.py

```diff
@@ -508,3 +514,38 @@ def apply_to_redis(self, host: str, port: int, password: str) -> None:
         for user in current_users:
             if user not in self.users:
                 db.acl_deluser(user)
+
+
+class AppYaml:
```

Comment:
it seems like this class could have some overlap / utility for the configfile part of the code. @koning , thoughts about how we could use this to refactor?

Reply:
Most of the server config can come from the app.yaml (username, passwords, port , server_type). we may be able to just add the rest of the container config to the server (broker , results_backend) sections.

Reply:
The container config corresponds to the [merlin_server.yaml](https://github.com/LLNL/merlin/blob/develop/merlin/server/merlin_server.yaml) config file where it details all of the files and configurations needed to be changed for the server container.

Reply:
Ok let's write down an issue to see how we can unify this with the configfile code